app/javascript/pages/RentDirectory.tsx

```diff
@@ -38,8 +38,7 @@ const getForRentSummaryTable = (listing: RailsRentalListing) =>
     .filter((summary) => !!summary.unitType)
     .map((summary) => ({
       unitType: {
-        cellText:
-          emptyIfNotTranslated(`listings.unitTypes.${summary.unitType}`) || summary.unitType,
+        cellText: emptyIfNotTranslated(`listings.unitTypes.${summary.unitType}`, summary.unitType),
```

Comment:
this looks like a typo maybe, needs to be renamed to defaultIfNotTranslated?

Reply:
oop, thanks for catching. still getting the hang of vscode and things like...saving files 😬